lib/_http_server.js

```diff
@@ -180,10 +180,20 @@ ServerResponse.prototype.detachSocket = function detachSocket(socket) {
 };
 
 ServerResponse.prototype.writeContinue = function writeContinue(cb) {
-  this._writeRaw(`HTTP/1.1 100 Continue${CRLF}${CRLF}`, 'ascii', cb);
+  this._writeInformation(100, 'Continue', cb);
   this._sent100 = true;
 };
 
+ServerResponse.prototype.writeProcessing = function writeProcessing(cb) {
+  this._writeInformation(102, 'Processing', cb);
+};
+
+ServerResponse.prototype._writeInformation = _writeInformation;
+function _writeInformation(statusCode, statusMessage, cb) {
+  this._writeRaw(`HTTP/1.1 ${statusCode} ${statusMessage}${CRLF}${CRLF}`,
+                 'ascii', cb);
+}
```

Comment:
can you make `writeInformation` as a top level function? There is no need to attach it to the prototype.
Another option is to just call `_writeRaw` in both cases, I do not see much harm in that duplication.

Reply:
Done.